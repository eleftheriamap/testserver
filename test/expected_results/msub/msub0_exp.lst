
test/exp/test/gen/msub/msub0_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d16c9718 	sub	x24, x24, #0xb25, lsl #12
   4:	915290a5 	add	x5, x5, #0x4a4, lsl #12
   8:	510b6694 	sub	w20, w20, #0x2d9
   c:	1b05d31e 	msub	w30, w24, w5, w20
  10:	8a000000 	and	x0, x0, x0
