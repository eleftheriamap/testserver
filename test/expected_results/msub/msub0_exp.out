Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 00000000004a4000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 00000000fffffd27
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = ffffffffff4db000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 00000000b3fffd27
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd16c9718
0x00000004 : 0x915290a5
0x00000008 : 0x510b6694
0x0000000c : 0x1b05d31e
0x00000010 : 0x8a000000
